node/cn/tracers/api.go

```diff
@@ -901,8 +901,8 @@ func (api *CommonAPI) traceTx(ctx context.Context, message blockchain.Message, b
 			}
 		}
 
-		if *config.Tracer == fastCallTracer {
```

Comment:
Minor, but const fastCallTracer can be removed if unused.

Reply:
Let me clean that up in later PRs. I kept them just in case we revert to InternalTxTracer.

Reply:
Better to remove unsed `InternalTxTracer`

Reply:
Okay. I'll make sure to remove that when v1.0.1 CallTrace is deemed stable.